var/spack/repos/builtin/packages/sublime-text/package.py

```diff
@@ -39,8 +40,12 @@ def url_for_version(self, version):
             return "https://download.sublimetext.com/sublime_text_{0}_build_{1}_x64.tar.bz2".format(version[0], version[-1])
 
     def install(self, spec, prefix):
-        # Sublime text comes as a pre-compiled binary.
         install_tree('.', prefix)
+        src = join_path(prefix, 'sublime_text')
+        dst = join_path(prefix, 'bin')
+        mkdirp(dst)
+        force_symlink(src, join_path(dst, 'sublime_text'))
+        force_symlink(src, join_path(dst, 'subl'))
 
     def setup_environment(self, spack_env, run_env):
```

Comment:
This method is no longer needed, spack adds `prefix.bin` to the `PATH` by default.

Reply:
I removed it.